cppsdk/include/playfab/PlayFabEvent.h

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#ifndef DISABLE_PLAYFABENTITY_API
```

Comment:
>DISABLE_PLAYFABENTITY_API [](start = 8, length = 25)

What is this ?

Reply:
It's the way PlayFab event API is enabled/disabled:
[https://github.com/PlayFab/XPlatCppSdk/blob/master/cppsdk/include/playfab/PlayFabAuthenticationDataModels.h#L3](https://github.com/PlayFab/XPlatCppSdk/blob/master/cppsdk/include/playfab/PlayFabAuthenticationDataModels.h#L3)

If we do not honor that macro, our code will break compilation because it relies on PlayFabEventDataModels.h which will be excluded from compilation in that case

---
In reply to: [227943601](https://github.com/PlayFab/XPlatCppSdk/pull/6#discussion_r227943601) [](ancestors = 227943601)